test/e2e/template/helmfile/snapshot_test.go

```diff
@@ -106,12 +106,15 @@ func TestHelmfileTemplateWithBuildCommand(t *testing.T) {
 					hostPort = 5000
 				}
 
-				execDocker(t, "run", "-d", "-p", fmt.Sprintf("%d:5000", hostPort), "--restart=always", "--name", containerName, "registry:2")
+				execDocker(t, "run", "--rm", "-d", "-p", fmt.Sprintf("%d:5000", hostPort), "--name", containerName, "registry:2")
 				t.Cleanup(func() {
 					execDocker(t, "stop", containerName)
-					execDocker(t, "rm", containerName)
 				})
 
+				// FIXME: this is a hack to wait for registry to be up and running
+				// please replace with proper wait for registry
+				time.Sleep(5 * time.Second)
```

Comment:
@stek29 I think we should write a checkport func to check. WDYT?

Reply:
I agree, but maybe it should be done in a separate PR?